Dont build with -Werror like switches.
Need to patch Makefile and *NOT* Makefile.in since otherwise this triggers a
rebuild of the Makefile which needs htmlc to be installed first.
https://bugs.gentoo.org/show_bug.cgi?id=553172

Index: htmlc-2.6.0/config/Makefile
===================================================================
--- htmlc-2.6.0.orig/config/Makefile
+++ htmlc-2.6.0/config/Makefile
@@ -40,7 +40,7 @@ MANDIR = $(PREFIXINSTALLDIR)/man/man$(MA
 
 # The Caml compilers (those defaults should be OK)
 CAMLDEP = $(OCAML_COMMAND_SEARCH_PATH)ocamldep
-CAML_FLAGS = -w A -warn-error A #-safe-string
+CAML_FLAGS = -w A #-safe-string
 CAMLBYT_FLAGS = $(CAML_FLAGS) -g -annot
 CAMLBYT = $(OCAML_COMMAND_SEARCH_PATH)ocamlc $(CAMLBYT_FLAGS)
 CAMLBIN_FLAGS = $(CAML_FLAGS) -inline 10000
